src/prototypes/Student.prototype.js

```diff
@@ -0,0 +1,25 @@
+module.exports = class Student {
+  constructor(idStudent, emailStudent, roleStudent, firstNameStudent, lastNameStudent, classStudent) {
+    this.id = idStudent;
+    this.email = emailStudent;
+    this.role = roleStudent;
+    this.firstName = firstNameStudent;
+    this.lastName = lastNameStudent;
+    this.class = classStudent;
+  }
+
+  static dbToStudent(obj) {
+    const course = new Student(obj.idStudent, obj.emailStudent, obj.roleStudent, obj.firstNameStudent, obj.lastNameStudent, obj.classStudent);
```

Comment:
In the database the attributes are all lowercase. You should use idstudent, emailstudent, etc.

Reply:
```
const course = new Student(obj.idstudent, obj.emailstudent, obj.rolestudent, obj.firstnamestudent, obj.lastnamestudent, obj['class-student']);
```